references/cicd-sharedflow-pipeline/test/integration/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "Test-SF-Security-v1",
+	"version": "1.0.0",
+	"description": "Sample Sharedflow for CI/CD Reference",
+	"author": "ssvaidyanathan",
+	"license": "Apache-2.0",
+	"licenses": [
+		{
+			"type": "Apache-2.0",
+			"url": "http://www.apache.org/licenses/LICENSE-2.0"
+		}
+	],
+	"scripts": {
+		"eslint": "eslint -c ./.eslintrc-jsc.yml --format json",
+		"apigeelint": "apigeelint -s ./sharedflowbundle -f table.js",
+		"lint": "npm run eslint && npm run apigeelint",
+		"integration-test": "cucumber-js ./target/test/integration"
+	},
+	"devDependencies": {
+		"prettyjson": ">=1.1.3",
```

Comment:
i dont think we need two `package.json` s for this project, but we can just use one at the project root

Reply:
Just wanted to keep them isolated as the package.json for the apiproxy has more dependencies.

Reply:
i think the increased cost of maintenance of two `package.json`s isnt worth the edge case that somebody wants to run the tests without deploying. in addition it isn't consistent with other devrel projects

Reply:
If we decide to use 2 poms, can we use two package.json files or should I still merge them?

Reply:
for this project, we only need one pom and one `package.json`. I believe nested node projects, multi-module poms and anything else that increases the complexity is unnecessary to demonstrate shared flow automation. @danistrebel @seymen for second opinion if needed